projects/frankencoin/index.js

```diff
@@ -1,14 +1,21 @@
-const { sumTokens2 } = require('../helper/unwrapLPs')
-const { cachedGraphQuery } = require('../helper/cache')
+const { sumTokens2 } = require('../helper/unwrapLPs');
+const { graphQuery } = require('../helper/http');
 
-const MintingHub = "0x7bbe8F18040aF0032f4C2435E7a76db6F1E346DF";
-const Collaterals = ["0xb4272071ecadd69d933adcd19ca99fe80664fc08"]; // XCHF
+// @dev: mapping of XCHF to its Bridge
+const XCHFBridge = ["0xb4272071ecadd69d933adcd19ca99fe80664fc08", "0x7bbe8F18040aF0032f4C2435E7a76db6F1E346DF"];
 
 async function tvl(api) {
-  const tokensAndOwners = Collaterals.map(i => [i, MintingHub])
-  const { positions } = await cachedGraphQuery('frankencoin', 'https://api.thegraph.com/subgraphs/name/frankencoin-zchf/frankencoin-subgraph', '{ positions { position collateral } }')
-  positions.forEach(i => tokensAndOwners.push([i.collateral, i.position]))
-  return sumTokens2({ api, tokensAndOwners, })
+  const tokensAndOwners = [XCHFBridge];
+
+  // @dev: query of positions from minting hubs via frankencoin graph (ponder)
+  const { positionV1s } = await graphQuery('https://ponder.frankencoin.com/', '{ positionV1s { items { position collateral } } }');
```

Comment:
can we use cachedGraphQuery() here instead?

Reply:
sure. adjusted those changes.